pymilvus/client/grpc_handler.py

```diff
@@ -84,6 +91,7 @@ def __init__(
         self._log_level = None
         self._request_id = None
         self._user = kwargs.get("user", None)
+        self._compression_type = kwargs.get("compression_type", "none")
```

Comment:
Please check if compression_type is valid and raise meaningful exception, or the following dict[xxx] might raise KeyError exception